src/lang/modifyAst/addRevolve.ts

```diff
@@ -67,34 +71,19 @@ export function revolveSketch(
   if (err(tagResult)) return tagResult
   const { tag } = tagResult
 
-  /* Original Code */
-  const { node: sketchExpression } = sketchNode
-
-  // determine if sketchExpression is in a pipeExpression or not
-  const sketchPipeExpressionNode = getNodeFromPath<PipeExpression>(
-    clonedAst,
-    pathToSketchNode,
-    'PipeExpression'
-  )
-  if (err(sketchPipeExpressionNode)) return sketchPipeExpressionNode
-  const { node: sketchPipeExpression } = sketchPipeExpressionNode
-  const isInPipeExpression = sketchPipeExpression.type === 'PipeExpression'
-
   const sketchVariableDeclaratorNode = getNodeFromPath<VariableDeclarator>(
     clonedAst,
     pathToSketchNode,
     'VariableDeclarator'
   )
   if (err(sketchVariableDeclaratorNode)) return sketchVariableDeclaratorNode
-  const {
-    node: sketchVariableDeclarator,
-    shallowPath: sketchPathToDecleration,
-  } = sketchVariableDeclaratorNode
+  const { node: sketchVariableDeclarator } = sketchVariableDeclaratorNode
 
   const axisSelection = axis?.graphSelections[0]?.artifact
 
   if (!axisSelection) return new Error('Axis selection is missing.')
 
+  // here
```

Comment:
Nit: Not sure what this comment means

Reply:
Oh all good, I don't want anyone to think they can't merge my PR if they look good to go.

Maybe just the size of it, and merging it on my Saturday it should have waited till start of the week. but it's also my bad because there should have been more test, I think I was a bit foggy when I came up with the original list of things to test.